components/ModEditFacilityOrProfessionalTopbar.vue

```diff
@@ -22,6 +22,16 @@
             </button>
         </div>
         <div class="facility-hp-topbar-actions flex justify p-2 font-bold ">
+            <button
+                type="button"
+                class="flex justify-center items-center rounded-full bg-secondary-bg border-primary-text-muted
+                border-2 w-28 text-sm mr-2"
+                data-testid="mod-edit-facility-hp-topbar-delete"
```

Comment:
It looks like you may have forgotten to update the `data-testid`. Since this is a "Back" button instead of a "Delete" button, what do you think about something like:

```suggestion
                data-testid="mod-edit-facility-hp-topbar-back-btn"
```

Reply:
@NabbeunNabi It looks like the same was done in PR #1148 that was merged and it'll need to be fixed.